prov/rxm/src/rxm_ep.c

```diff
@@ -270,8 +270,10 @@ static int rxm_ep_getopt(fid_t fid, int level, int optname, void *optval,
 
 	switch (optname) {
 	case FI_OPT_MIN_MULTI_RECV:
-		return rxm_ep->srx->ep_fid.ops->getopt(&rxm_ep->srx->ep_fid.fid,
-						level, optname, optval, optlen);
+		assert(sizeof(rxm_ep->min_multi_recv_size) == sizeof(size_t));
+		*(size_t *)optval = rxm_ep->min_multi_recv_size;
+		*optlen = sizeof(size_t);
+		break;
```

Comment:
Both situations need to be handled, for example:
```
        if (rxm_ep->srx) {
                /* the old code here */
        } else {
                /* the new code here */
        }
```

Reply:
ok fixed